doc/modules/mod_event_pusher_sns.md

```diff
@@ -0,0 +1,59 @@
+### Module Description
+
+This module is a backend of [mod_event_pusher] that enables support for the Amazon SNS service. Currently there are 3 available notifications:
+
+* **user presence changed** - Carries the user id (bare jid by default) and a boolean field corresponding to the current user online status.
+* **private message sent** - Carries the user ids (both sender and receiver) along with the message body.
+* **group message sent** - Carries the user id and the room id (bare jids by default) along with the message body.
+
+All those notifications are sent as a JSON string to Amazon SNS along with custom MessageAttributes (see http://docs.aws.amazon.com/sns/latest/api/API_Publish.html). MessageAttributes can be specified via a plugin module (more details in *Options* section).
```

Comment:
All these notifications are sent as a JSON string to Amazon SNS along with custom MessageAttributes (see http://docs.aws.amazon.com/sns/latest/api/API_Publish.html). 
MessageAttributes can be specified via a plugin module (more details in *Options* section).